examples/pubsub_realtime/TSN_ETF_XDP_publisher.c

```diff
@@ -40,10 +40,17 @@
 #include <open62541/plugin/log.h>
 #include <open62541/types_generated.h>
 #include <open62541/plugin/pubsub_ethernet.h>
+#ifdef UA_ENABLE_PUBSUB_ETH_UADP_ETF
```

Comment:
XDP and ETF are independent.
ETF works on all RT Linux, XDP depends on specific versions.

Making ETF depend on XDP for the publisher is a regression.
Please don't.